app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+
+  belongs_to :user
+  has_many :likes
+  has_many :users, through: :likes
```

Comment:
То же самое, что и в `user.rb`
```suggestion
  has_many :liked_users, through: :likes, source: :user
```